model/export/SingleDeliveryResultsExporter.php

```diff
@@ -138,19 +115,15 @@ public function getResourceToExport()
      */
     public function setColumnsToExport($columnsToExport)
```

Comment:
suggestion: you removed is_string check then probably you can define array type for $columnsToExport